cards/en/sm10.json

```diff
@@ -11297,6 +11297,82 @@
       "large": "https://images.pokemontcg.io/sm10/195_hires.png"
     }
   },
+  {
+    "id": "sm10-195a",
```

Comment:
We could either append `a` to the non-alternate Dedenne GX or increase by 1 (sm10-196) and increase all ids below. Increasing, however, could break many clients that are relying on the ids for tracking or any other feature and so I believe appending `a` to the original prefix is the best option here.

Reply:
Yes, appending the `a` is how it has been done with other cards, so that looks good.